src/components/CreateSighting.tsx

```diff
@@ -0,0 +1,114 @@
+import React from "react"; 
+import { useState } from "react";
+import { FormEvent } from "react";
+import './CreateSighting.scss';
+
+export function CreateSighting() {
+    const [date, setDate] = useState<string>("");
+    const [photoUrl, setPhotoUrl] = useState<string>("");
+    const [latitude, setLatitude] = useState<number>(0);
+    const [longitude, setLongitutde] = useState<number>(0);
+    const [numberOfWhales, setNumberOfWhales] = useState<number>(0);
+    const [species, setSpecies] = useState<string>("");
+    const [description, setDescription] = useState<string>("");
+
+    function handleSubmit(event: FormEvent<HTMLFormElement>) {
+        event.preventDefault()
+    }
+
+    return (
+        <main className="create-sighting">
```

Comment:
We can remove the brackets here by moving the opening <main> tag onto the same line as the return statement
```suggestion
    return <main className="create-sighting">
```